XamarinCommunityToolkit/Views/CameraView/CameraViewRenderer.uwp.cs

```diff
@@ -0,0 +1,399 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Windows.Devices.Enumeration;
+using Windows.Devices.Lights;
+using Windows.Graphics.Display;
+using Windows.Graphics.Imaging;
+using Windows.Media.Capture;
+using Windows.Media.Core;
+using Windows.Media.Devices;
+using Windows.Media.MediaProperties;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.System.Display;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.UWP;
+using MediaSource = Xamarin.Forms.MediaSource;
+
+[assembly: ExportRenderer(typeof(CameraView), typeof(CameraViewRenderer))]
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class CameraViewRenderer : ViewRenderer<CameraView, CaptureElement>
+	{
+		MediaCapture mediaCapture;
+		bool isPreviewing;
+		Lamp flash;
+		LowLagMediaRecording mediaRecording;
+		string filePath;
+		bool busy;
+		VideoStabilizationEffect videoStabilizationEffect;
+		MediaEncodingProfile encodingProfile;
+		VideoEncodingProperties inputPropertiesBackup;
+		VideoEncodingProperties outputPropertiesBackup;
+
+		public CameraViewRenderer()
+		{
+			encodingProfile = MediaEncodingProfile.CreateMp4(VideoEncodingQuality.Auto);
+		}
+
+		bool IsBusy
+		{
+			get => busy;
+			set
+			{
+				if (busy != value)
+					Element.IsBusy = busy = value;
+			}
+		}
+
+		bool Available
```

Comment:
I realized that we just set this property, but never use it to verify anything. Do you have a guess where we want to check if the camera is Available?

Reply:
We can remove it if it's not used.  I'll double check that.

Reply:
I wonder if it at one point it was used like IsBusy, but it's not used now.  Removed

Reply:
Actually that was used to set the IsAvailable on the CameraView so I added it back.

Reply:
I missed that, I'm sorry 🙈